pixel-point/bytebase-typescript-next
# feat: develop changelog

This pull request brings changelog:

**Steps to check:**
- review [filtered code](https://github.com/pixel-point/bytebase-typescript-next/pull/37/files?file-filters%5B%5D=.css&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx&show-viewed-files=true) (without md and images taken from current site)
- open [preview](https://bytebase-typescript-next-git-changelog-bytebase-pixelpoint.vercel.app/changelog/)
- make sure that everything looks and feels as expected
